src/components/ClusterNodes/ClusterEvents.tsx

```diff
@@ -15,12 +22,13 @@ export default function ClusterEvents({ terminalAccessId }: { terminalAccessId:
             getClusterEvents(terminalAccessId)
                 .then((response) => {
                     setLoading(false)
-                    const events = response.result?.events.items
+                    const events = response.result
                     setEvents(events)
```

Comment:
we can directly set the data into setEvents no need to create the variable. also please accomodatae `events?.eventsResponse?.events.items` it here only and and not at the time of passing the props on line no 45